torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -996,20 +996,20 @@ void ProcessGroupNCCL::broadcastUniqueNCCLID(
   // retrieving the contents of that key. A single process group
   // may create multiple NCCL communicators, so we use a sequence
   // number to differentiate between them.
-  // For point-to-point operations:
+  // For single point-to-point operations:
```

Comment:
just for my understanding, does single == non-batched api version?

Reply:
Yes, that's right :)